R/json_test.R

```diff
@@ -120,7 +122,7 @@ json_test <- function(
       obj <- parse_json_test(sel_tests[[key]], parse_functions)
       time_a <- Sys.time()
       if(list_as_args) {
-        do.call(what = "fnc", args = obj)
+        tmp <- do.call(what = "fnc", args = obj)
```

Comment:
I think this was an omission since otherwise the results of the do.call are never saved.

IMO we should find a way to test the function without requiring the function be defined with an argument called args.